package.json

```
@@ -25,13 +25,13 @@
   },
   "peerDependencies": {
     "@alaskaairux/auro-icon": "^3.3.1",
-    "@alaskaairux/design-tokens": "^3.11.0",
-    "@aurodesignsystem/webcorestylesheets": "^4.2.0"
+    "@aurodesignsystem/design-tokens": "^4.1.1",
```

Both should be listed for now.
```suggestion
    "@alaskaairux/design-tokens": "^3.11.0",
    "@aurodesignsystem/design-tokens": "^4.1.1",
```